packages/formation/.npmignore

```diff
@@ -1,2 +1,3 @@
 scripts/*
-assets/*
+assets/img/docs/*
+assets/img/design/*
```

Comment:
updated ignores used during publishing

Reply:
Could we delete them from the repo, too?